api/Controllers/SuppliersController.cs

```diff
@@ -12,6 +13,33 @@ public SuppliersController(SupplierProvider supplierProvider)
         _supplierProvider = supplierProvider;
     }
 
+    [HttpGet("{id}/items")]
+    public IActionResult GetItemsSupplier(Guid id)
+    {
+        var items = _supplierProvider.GetItemsBySupplierId(id);
+        if (items == null) throw new ApiFlowException("this supplier does not have any items yet");
+
+        var itemsSupplier = items.Select(item => new ItemResponse
+        {
+            Id = item.Id,
+            Code = item.Code,
+            Description = item.Description,
+            ShortDescription = item.ShortDescription,
+            UpcCode = item.UpcCode,
+            ModelNumber = item.ModelNumber,
+            CommodityCode = item.CommodityCode,
+            UnitPurchaseQuantity = item.UnitPurchaseQuantity,
+            UnitOrderQuantity = item.UnitOrderQuantity,
+            PackOrderQuantity = item.PackOrderQuantity,
+            SupplierReferenceCode = item.SupplierReferenceCode,
+            SupplierPartNumber = item.SupplierPartNumber,
+            ItemGroupId = item.ItemGroupId,
+            SupplierId = item.SupplierId,
+        }).ToList();
+
+        return Ok(itemsSupplier);
```

Comment:
Je hoeft hiervoor niet een var te maken het kan in de Response

Reply:
bro je kan een supplier vinden, maar als die supplier geen items heeft, dan moet ie dat wel laten zien